docs/general/polkadot-v2.md

```diff
@@ -0,0 +1,234 @@
+---
+id: polkadot-v2
+title: Potential Changes to Polkadot
+sidebar_label: Potential Changes to Polkadot
+description: Polkadot as a Decentralized and Secure Ubiquitous Computer.
+keywords:
+  [polkadot vison, polkadot 2.0, Decentralization, ubiquitous computer, coretime, blockspace]
+slug: ../polkadot-v2
```

Comment:
Just to reiterate what we've discussed in the Marcomms channel: we should avoid any public mentions of "Polkadot 2.0" and instead always specifically talk about the name of the feature that we want to explain.